cubi_tk/sodar/pull_raw_data.py

```diff
@@ -189,6 +203,12 @@ def _build_commands(self, assay, library_to_folder):
             ]
         return commands
 
+    def _build_check_commands(self, assay, library_to_folder):
```

Comment:
Please keep code documented. Example:
```
"""Method summary.

:param assay: Assay definition.
:type assay: Assay type.

...

:returns: Explanation of what the method returns.
"""
```

Also, can't this be a `@staticmethod`?